### PR TITLE
Add "nuke user" button to user permissions page

### PR DESCRIPTION
**Pull request by NobodyNada**
This PR adds a "nuke" button to the right of the checkboxes on the user permissions page, to allow admins to easily destroy users.  The button uses AJAX and removes the row from the table on success.  It asks for confirmation the first time it's used, but after you've nuked one user it won't ask for confirmation until you refresh the page.

The code probably has room for improvement, since I don't do frontend very much.

### CODE REVIEW

File: app/views/admin/permissions.html.erb [hunk 1]
```diff
@@ -33,6 +34,7 @@
           </label>
         </td>
       <% end %>
+      <td><%= link_to "Nuke", 'javascript:void(0);', class: 'text-danger nuke-user-link', data: { :user_id => user.id, :username => user.username } %></td>
```
Comment (ArtOfCode-) [inline app/views/admin/permissions.html.erb]: Can we wrap this in a `<% if current_user&.has_role?(:developer) %>` to restrict it down to developers for now please?

**Comment by ArtOfCode-** [comment]
Have you tested?

File: app/views/admin/permissions.html.erb (hunk 1 above)
Comment (NobodyNada) [reply app/views/admin/permissions.html.erb]: [status-completed]; `verify_developer` added to controller as well.

**Comment by NobodyNada** [comment]
I have tested.

**Pull request merged**